app/services/clients/base-client/base.client.js

```diff
@@ -36,3 +35,68 @@ module.exports = {
 function retryStrategy (err, response, body) {
   return process.env.NODE_ENV === 'production' && (err && ['ECONNRESET'].includes(err.code))
 }
+
+function makeRequest (method, opts) {
+  return new Promise((resolve, reject) => {
+    opts.method = method.toUpperCase()
+
+    const loggingContext = {
+      correlationId: correlator.getId(),
```

Comment:
🤔 if we do this on a base client and have [middleware to set this up on every request](https://github.com/alphagov/pay-selfservice/blob/master/app/routes.js#L152) why do we have any correlation IDs being passed around through code? 

I don't think these apps do anything outside of the context of responding to HTTP requests.

Reply:
Oh yeh good point, I'll look a bit more into what's going on here. I'll merge this one as this functionality hasn't changed, and raise a new PR if appropriate.